core/src/main/scala/org/allenai/common/Version.scala

```diff
@@ -81,19 +83,18 @@ object Version {
     val pkg = "/" + org + "/" + name.replaceAll("-", "")
     val artifactConfUrl = this.getClass.getResource(pkg + "/artifact.conf")
     val gitConfUrl = this.getClass.getResource(pkg + "/git.conf")
-
+    val cacheKeyConfUrl = this.getClass.getResource(pkg + "/cacheKey.conf")
```

Comment:
Doesn't the deployplugin have 2 stage steps even without cacheKey generation?
// Clean up anything leftover in the staging directory before re-staging.
    UniversalPlugin.autoImport.stage <<=
      UniversalPlugin.autoImport.stage.dependsOn(cleanStage),